docs/source/api/v3/staticdnsentries.rst

```diff
@@ -234,7 +234,7 @@ Request Structure
 	|  id  | The integral, unique identifier of the static DNS entry to modify |
 	+------+-------------------------------------------------------------------+
 
-:address:           If ``typeId`` identifies a ``CNAME`` type record, this is the Canonical Name (CNAME) of the server, otherwise it is the IP address to which ``host`` shall be resolved
+:address:           If ``typeId`` identifies a ``CNAME`` type record, this is the Canonical Name (CNAME) of the server that is required to end with a trailing period (eg:cdn.test.com. is right usage but cdn.test.com will throw an error), otherwise it is the IP address to which ``host`` shall be resolved
```

Comment:
again. make the descriptions consistent.

Reply:
> also, there are 4 types, right?
> 
> ![image](https://user-images.githubusercontent.com/251272/89673576-d4391100-d8a3-11ea-90a1-e946a0ca5b00.png)
> 
> are there any rules for TXT_RECORD? if so, want to add it to your tooltip?
> 
> ![image](https://user-images.githubusercontent.com/251272/89673532-c7b4b880-d8a3-11ea-9629-fca88d1ba1fe.png)

Code updated.